draft-ietf-tls-tls13.md

```diff
@@ -313,6 +313,11 @@ server: The endpoint which did not initiate the TLS connection.
 ##  Major Differences from TLS 1.2
 
 
+draft-09
+
+- Change to RSA-PSS signatures.
```

Comment:
Just to be clear what is changed:
"signatures" -> "signatures for handshake messages"